### ISSUE TITLE
dotenv variables not available after update

### ISSUE THREAD

**Issue by FezVrasta** [timarney/react-app-rewired#154]
Hi, I updated my project dependencies (`yarn upgrade`) and now it looks like dotenv is not populating the `process.env` variables in the modules I import, for instance:

```js
// config-overrides.js

const rewireWebpack = require('@private/rewire-webpack');
module.exports = {
  webpack(config, env) {
    console.log(process.env.NODE_PATH); // Here it is correctly defined
    return compose(
      config => rewireWebpack(config, env),
    )(config);
  },
};
```

```js
// @private/rewire-webpack

module.exports = (config, env) => ({
  ...config,
  something: process.env.NODE_PATH, // Here is not defined
}
```

This worked just fine before I upgraded the dependencies, I wonder what changed to break it?

As workaround, if I run the scripts with `node -r dotenv/config node_modules/react-app-rewired/scripts/start.js` the environment variables are properly populated.

The problem seem to have been introduced in 1.3.6, with 1.3.5 everything is fine

**Comment by timarney** [comment]
@Guria can you take a look at this - related to https://github.com/timarney/react-app-rewired/pull/152 by the sounds of it.

**Comment by Guria** [comment]
Sounds weird for me. I double checked on my setup that dotenv applied correctly.
Though I believe I am missing smth here.
Will check it again.

@FezVrasta could you publish reproducible repository on github?

**Comment by Guria** [comment]
Ok. I have an idea now. Seems you report was enough. Will prepare a fix soon.

**Comment by Guria** [comment]
Sorry for delay. Will make a fix soon

**Comment by Guria** [comment]
@FezVrasta Can you try to apply a patch from #160 directly in node_modules folder and report back a result?

**Comment by FezVrasta** [comment]
Hey, sure, I'll check it out on Monday

**Comment by timarney** [comment]
published react-app-rewired@1.3.7
